src/operation/mod.rs

```diff
@@ -153,6 +153,9 @@ pub(crate) trait Response: Sized {
     /// The `atClusterTime` field of the response.
     fn at_cluster_time(&self) -> Option<Timestamp>;
 
+    /// The `recoveryToken` field of the response.
+    fn recovery_token(&self) -> Option<Document>;
```

Comment:
For consistency with the other methods in this trait, can this be updated to return `Option<&Document>` instead? This will also help prevent any accidental over-cloning in the future.